src/Graphs.jl

```diff
@@ -10,162 +10,409 @@ using Statistics: mean
 using Compat
 
 using Inflate: InflateGzipStream
-using DataStructures: IntDisjointSets, PriorityQueue, dequeue!, dequeue_pair!, enqueue!, heappop!, heappush!, in_same_set, peek, union!, find_root!
+using DataStructures:
```

Comment:
The huge diff is due to exports being line by line, following blue style specification